org.adempiere.ui.zk/WEB-INF/src/web/js/org/idempiere/commons/report.js

```diff
@@ -3,38 +3,38 @@ if (typeof window.idempiere === 'undefined')
 	
 window.idempiere.zoom = function(cmpid, column, value){
 	zAu.cmd0.showBusy(null);
-	var widget = zk.Widget.$(cmpid);
-	var event = new zk.Event(widget, 'onZoom', {data: [column, value]}, {toServer: true});
+	let widget = zk.Widget.$(cmpid);
+	let event = new zk.Event(widget, 'onZoom', {data: [column, value]}, {toServer: true});
 	zAu.send(event);
 };
 
 window.idempiere.zoomWindow = function(cmpid, column, value, windowuu){
 	zAu.cmd0.showBusy(null);
-	var widget = zk.Widget.$(cmpid);
-	var event = new zk.Event(widget, 'onZoom', {data: [column, value, 'AD_Window_UU', windowuu]}, {toServer: true});
+	let widget = zk.Widget.$(cmpid);
+	let event = new zk.Event(widget, 'onZoom', {data: [column, value, 'AD_Window_UU', windowuu]}, {toServer: true});
 	zAu.send(event);
 };
 
 window.idempiere.drillAcross = function(cmpid, column, value){
 	zAu.cmd0.showBusy(null);
-	var widget = zk.Widget.$(cmpid);
-	var event = new zk.Event(widget, 'onDrillAcross', {data: [column, value]}, {toServer: true});
+	let widget = zk.Widget.$(cmpid);
+	let event = new zk.Event(widget, 'onDrillAcross', {data: [column, value]}, {toServer: true});
 	zAu.send(event);
 };
 
 window.idempiere.drillDown = function(cmpid, column, value){
 	zAu.cmd0.showBusy(null);
-	var widget = zk.Widget.$(cmpid);
-	var event = new zk.Event(widget, 'onDrillDown', {data: [column, value]}, {toServer: true});
+	let widget = zk.Widget.$(cmpid);
+	let event = new zk.Event(widget, 'onDrillDown', {data: [column, value]}, {toServer: true});
 	zAu.send(event);
 };
 
 window.idempiere.showColumnMenu = function(doc, e, columnName, row) {
-	var d = idempiere.getMenu (doc, e.target.getAttribute ("componentId"), e.target.getAttribute ("foreignColumnName"), e.target.getAttribute ("value"));
+	let d = idempiere.getMenu (doc, e.target.getAttribute ("componentId"), e.target.getAttribute ("foreignColumnName"), e.target.getAttribute ("value"));
 	
-	var posx = 0;
-	var posy = 0;
-	if (!e) var e = window.event;
+	let posx = 0;
+	let posy = 0;
+	if (!e) let e = window.event;
```

Comment:
I got warning about **let** here. Should this be:
`if (!e) e = window.event;` ?